packages/app-generic-asset/src/Transfer.tsx

```diff
@@ -48,6 +53,19 @@ function Transfer ({ assets, className, onClose, recipientId: propRecipientId, s
     })));
   }, [assets]);
 
+  useEffect((): void => {
+    if (assetBalance && !assetBalance.isZero()) {
```

Comment:
` setHasBalance(assetBalance && !assetBalance.isZero())`

same with the below
`setHasAvailable(amount && !amount.isZero())`